### PR TITLE
Blog post announcement salary survey 2021

### PR DESCRIPTION
**Pull request by thatdocslady**
Since it's already been two weeks since the newsletter went out (which means we only have a couple of weeks until the next one!), I propose this updated plan: 

1. Publish this blog post without mailchimp RSS and share the link on social media
2. When the next newsletter goes out mention the survey again and link to the survey directly

This will both distribute the load on the server and give us more opportunities for buzz, rather than hook into the newsletter mailing list out of schedule. Thoughts?

### CODE REVIEW

**Comment by ericholscher** [comment]
Plan makes sense -- we definitely shouldn't send just a salary survey note to the newsletter, but include it in an existing newsletter as a link.

File: docs/blog/2021-salary-survey.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.. post:: Oct 19, 2021
+   :tags: salary survey
```
Comment (ericholscher) [inline docs/blog/2021-salary-survey.rst]:
```suggestion
   :tags: salary-survey
```

File: docs/blog/2021-salary-survey.rst [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+Results will be published in early 2022. 
+Shout out to `Kay Smoljak <https://twitter.com/goatlady>`_ for running this survey since it debuted in 2019!
+
+Thanks
+------
+
+Thanks again for for being a member of our community.
+We hope to see you soon at one of our online events, on our slack, or continue to see you here via this newsletter.
+
+You can always reply to this email if you have any questions or comments.
```
Comment (ericholscher) [inline docs/blog/2021-salary-survey.rst]: This won't go out in email, right?

**Review by ericholscher — APPROVED** [review]
Simple enough 👍

File: docs/blog/2021-salary-survey.rst (hunk 2 above)
Comment (thatdocslady) [reply docs/blog/2021-salary-survey.rst]:
```suggestion
You can always email us at support@writethedocs.org if you have any questions or comments.
```

File: docs/blog/2021-salary-survey.rst [hunk 3]
```diff
@@ -20,7 +20,7 @@ We hope that the survey data can help our community members determine what appro
 
 Additionally, we’re hoping the results spark discussion of employment-related trends and issues in our industry, including but not limited to the effects of the COVID-19 pandemic.
 
-The survey is divided into 5 sections, and takes between 5 and 10 minutes to complete, so go ahead and `fill it out <https://salary-survey.writethedocs.org/>`__. 
+The survey is divided into 6 sections, and takes between 5 and 10 minutes to complete, so go ahead and `fill it out <https://salary-survey.writethedocs.org/>`__.
```
Comment (ericholscher) [inline docs/blog/2021-salary-survey.rst]: Numbers are bad..

```suggestion
The survey is divided into a few different sections, and takes between 5 and 10 minutes to complete, so go ahead and `fill it out <https://salary-survey.writethedocs.org/>`__. 
```

**Pull request merged**